modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -116,17 +220,43 @@ def predict(self, booster: xgb.Booster, **kwargs):
 
 
 def _get_cluster_cpus():
+    """Get number of CPUs available on Ray cluster.
+
+    Returns
+    -------
+    int
+        Number of CPUs available on cluster.
+    """
     return ray.cluster_resources().get("CPU", 1)
 
 
 def _get_min_cpus_per_node():
+    """Get min number of node CPUs available on cluster nodes.
+
+    Returns
+    -------
+    int
+        Min number of CPUs per node.
+    """
     max_node_cpus = min(
```

Comment:
name of variable conflicts with name of function :)

Reply:
Added TODO to fix in future.